forward_engineering/helpers/mapAvroSchema.js

```diff
@@ -11,9 +11,17 @@ const mapAvroSchema = (avroSchema, iteratee) => {
 
 	if (Array.isArray(avroSchema.fields)) {
 		const fields = avroSchema.fields.map(field => {
+			const typeSchema = mapAvroSchema(field.type, iteratee);
+			if (Array.isArray(typeSchema.type)) {
```

Comment:
nested union types are restricted in Avro, so we need to apply attributes changes to the field schema